src/UIHandler.cpp

```diff
@@ -0,0 +1,327 @@
+#include "UIHandler.h"
+
+UIWindow::UIWindow(int minSizeX, int minSizeY, Location location, bool drawBorder) : m_panel(new UIPaddingPanel(drawBorder))
+{
+    m_minSize.x = minSizeX;
+    m_minSize.y = minSizeY;
+
+    m_thisLocation = location;
+
+    UpdateWindow();
+}
+
+int UIWindow::UpdateWindow()
+{
+    // TODO: Clear prev contents? Is that our job?
```

Comment:
AH, I forgot to remove that. We do werase(m_wf_win) in UIWindow::DrawEverything()